### PR TITLE
[ut] increase covrage for kubelet/prober/results

### PR DESCRIPTION
**Pull request by my-git9**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Increase covrage for kubelet/prober/results

before:
```
        k8s.io/kubernetes/pkg/kubelet/prober/results    coverage: 69.2% of statements
```

after this change:
```
        k8s.io/kubernetes/pkg/kubelet/prober/results    coverage: 100.0% of statements
```

#### Which issue(s) this PR fixes:

Part of https://github.com/kubernetes/kubernetes/issues/109717

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
NONE
```

### CODE REVIEW

File: pkg/kubelet/prober/results/results_manager_test.go [hunk 1]
```diff
@@ -97,3 +97,65 @@ func TestUpdates(t *testing.T) {
 	m.Set(barID, Success, pod)
 	expectUpdate(Update{barID, Success, pod.UID}, "changed bar")
 }
+
+func TestResult_ToPrometheusType(t *testing.T) {
+	tests := []struct {
+		name string
+		r    Result
+		want float64
```
Comment (bart0sh) [inline pkg/kubelet/prober/results/results_manager_test.go:105]: nit: try to follow naming conventions used in this module, e.g. want -> expected, tt -> test etc.

File: pkg/kubelet/prober/results/results_manager_test.go [hunk 2]
```diff
@@ -97,3 +97,65 @@ func TestUpdates(t *testing.T) {
...
+
+func TestResult_ToPrometheusType(t *testing.T) {
+	tests := []struct {
+		name     string
+		r        Result
+		expected float64
+	}{
+		{
+			name:     "result is 0",
+			r:        0,
```
Comment (bart0sh) [inline pkg/kubelet/prober/results/results_manager_test.go:109]: I'd suggest to use constants here and in the name, e.g. name: "result is Success", r: Success, etc.

File: pkg/kubelet/prober/results/results_manager_test.go [hunk 3]
```diff
@@ -97,3 +97,65 @@ func TestUpdates(t *testing.T) {
 	m.Set(barID, Success, pod)
 	expectUpdate(Update{barID, Success, pod.UID}, "changed bar")
 }
+
+func TestResult_ToPrometheusType(t *testing.T) {
+	tests := []struct {
+		name     string
+		r        Result
```
Comment (bart0sh) [inline pkg/kubelet/prober/results/results_manager_test.go:104]: nit: I'd suggest to rename r -> result for more readability.

File: pkg/kubelet/prober/results/results_manager_test.go [hunk 4]
```diff
@@ -97,3 +97,65 @@ func TestUpdates(t *testing.T) {
...
+
+func TestResult_String(t *testing.T) {
+	tests := []struct {
+		name     string
+		result   Result
+		expected string
+	}{
+		{
+			name:     "result is Success",
+			result:   0,
```
Comment (bart0sh) [inline pkg/kubelet/prober/results/results_manager_test.go:140]: Can you use constants instead of their values here ? e.g. result: Success

**Review by mrunalp — APPROVED** [review]

**Pull request merged**